src/linear_samplers/block_col_SRHT.jl

```diff
@@ -0,0 +1,85 @@
+
+"""
+    LinSysBlockColSRHT <: LinSysBlkColSampler
+
+A mutable structure with fields to handle SRHT column sketching. For this procedure,
+the hadamard transform and random sign swaps are applied once, then that matrix is repeatably
+sampled.
```

Comment:
Same comment as before. Add an explanation of the terms and which ones are re-sampled and which ones are kept fixed.